src/components/AddObsModal/AddObsButton.js

```diff
@@ -2,25 +2,52 @@
 
 import { CommonActions, useNavigation } from "@react-navigation/native";
 import AddObsModal from "components/AddObsModal/AddObsModal.tsx";
-import { Modal } from "components/SharedComponents";
+import { Body2, Modal } from "components/SharedComponents";
 import GradientButton from "components/SharedComponents/Buttons/GradientButton.tsx";
 import { t } from "i18next";
 import { getCurrentRoute } from "navigation/navigationUtils.ts";
 import * as React from "react";
+import Tooltip from "react-native-walkthrough-tooltip";
 import { log } from "sharedHelpers/logger";
-import { useLayoutPrefs } from "sharedHooks";
-import useStore from "stores/useStore";
+import { useCurrentUser, useLayoutPrefs } from "sharedHooks";
+import useStore, { zustandStorage } from "stores/useStore";
 
 const logger = log.extend( "AddObsButton" );
 
-const AddObsButton = (): React.Node => {
+const AddObsButton = ( ): React.Node => {
   const [showModal, setModal] = React.useState( false );
 
   const openModal = React.useCallback( () => setModal( true ), [] );
   const closeModal = React.useCallback( () => setModal( false ), [] );
 
-  const resetObservationFlowSlice = useStore( state => state.resetObservationFlowSlice );
   const { isAllAddObsOptionsMode } = useLayoutPrefs( );
+  const currentRoute = getCurrentRoute( );
+  const currentUser = useCurrentUser( );
+
+  // Controls whether to show the tooltip, and to show it only once to the user
+  const showKey = "AddObsButtonTooltip";
+  const shownOnce = useStore( state => state.layout.shownOnce );
```

Comment:
Can we rename `shownOnce` to be more descriptive about which components are relying on this global state? I can't keep track, but I know it's at least pivot cards and this tooltip. Angie's dismissable banner PR is going to add a similar store called `loginBannerDismissed` which has slightly different viewing conditions but is also shown once, so I'd love to see a name like that: `pivotCardsAndToolTipShown` or something along those lines.